script.js

```diff
@@ -0,0 +1,164 @@
+//1
+const createTeam = (teamName, driverName, birthYear, car, sponsor, isAvailableDriver) => {
+    const team = {
+        teamName,
+        driverName,
+        birthYear,
+        car,
+        sponsor,
+        isAvailableDriver
+    };
+    return team
+}
+const myFirstTeam = createTeam('Rolling Stones', 'Ken Block', '1984', 'Ford', 'Red Bull', true)
+//console.log(myFirstTeam)
+
+//2
+const createNewUser = () => {
+    const userNew = {
+      login: "",
+      password: "",
+      city: "",
+      country: "",
+      gender: "",
+      age: "",
+    };
+  
+    for (const [key] of Object.entries(userNew)) {
+      let result = prompt([key]);
+      if (result === null || result === "") {
+        userNew[key] = EMPTY_VALUE;
+      } else {
+        userNew[key] = result;
+      }
+    }
+    return userNew;
+  };
+  const newUser = createNewUser();
+
+const userLogin = prompt('Enter your login.');
+const userPassword = prompt('Enter your password.');
+const userCity = prompt('Enter your city.');
+const userCountry = prompt('Enter your country.');
+const userGender = prompt('Enter your gender.');
+const userAge = prompt('Enter your age.');
+const userData = {
+    userLogin,
+    userPassword,
+    userCity,
+    userCountry,
+    userGender,
+    userAge
+};
+
+const user2Login = prompt('Enter your login.');
+const user2Password = prompt('Enter your password.');
+const user2City = prompt('Enter your city.');
+const user2Country = prompt('Enter your country.');
+const user2Gender = prompt('Enter your gender.');
+const user2Age = prompt('Enter your age.');
+const userData2 = {
+    userLogin: user2Login,
+    userPassword: user2Password,
+    userCity: user2City,
+    userCountry: user2Country,
+    userGender: user2Gender,
+    userAge: user2Age
+};
+
+//3
+const changeUserData = (user, key, value) => {
+    user[key] = value
+}
+
+changeUserData(userData, 'userCity', 'Kherson')
+console.log(userData)
+changeUserData(userData2, 'userAge', '12')
+console.log(userData2)
+
+//4
+const student = {
+    studentName: 'Steve',
+    studentSurname: 'Sad',
+    studentAge: 18,
+    studentCourse: 3,
+    studentCity: 'Washington',
+    addHomework: () => {
+        console.log('Sending my howework... Please, wait');
+    },
+    greeting: () => {
+        console.log('Hi everyone!');
+    }
+}
+student.greeting();
+student.addHomework();
+
+//5
+const isEmpty = (obj) => {
+    const keys = Object.keys(obj);
+    if (keys.length === 0) {
+        return true
+    } else {
+        return false
+    }
```

Comment:
It can be shorter 
```suggestion
   return keys.length === 0) ? true : false
```

or even 

`return Object.keys(value).length === 0`